### DISCUSSION TITLE
Add currency information or show a single currency for amounts on payment requests overview

### DISCUSSION THREAD

**Discussion by jacobburrell** [btcpayserver/btcpayserver#4619] [Ideas / Feature Requests]
Add currency indicator to payment requests to amounts column or add the conversion to view all amounts as a single currency value

If your store makes payment requests in more than one type of fiat currency, it can be confusing to see a side-by-side comparisons of fiat amounts without being able to identify what currency it's specified in.

E.g.

<img width="153" alt="image" src="https://user-images.githubusercontent.com/61520543/217966730-27d9d624-5c78-49c2-8665-f8a794dec6c4.png">

While all these fiat currencies above use the `$` symbol, some of them are valued differently.
In this case, the top value is the equivalent of about $30 USD, but the bottom one is $500 USD.

At a glance, this seems to be misleading.

Either add the currency symbol, e.g. MX$ or MXN to specify which fiat currency it is, or allow a view with only one currency that auto-does the conversion using the exchange rate at the time of either creation or payment.

Additionally and separately, it would be great to add a total to the table, along with the running balance for paid / unpaid payments. requests with more advanced features to filter based on status, amount (over $50, under $10,000), currency, etc.

**Comment by bolatovumar** [comment]
@jacobburrell started a draft RP here for the currency code display issue: https://github.com/btcpayserver/btcpayserver/pull/4709

Not sure if this is the best way of dealing with this issue, would be interested to hear what the team thinks.

**Reply by bolatovumar** [reply]
Ok, PR is now merged.
